src/workspace-data/data-table/entity-service/EntitiesContent.js

```diff
@@ -507,17 +532,71 @@ const EntitiesContent = ({
   };
 
   const renderIGVMenu = () => {
-    return renderIconButton(
-      igvLogo,
-      !entitiesSelected ? 'Select rows to open in IGV' : 'Open with Integrative Genomics Viewer',
-      () => {
-        setSelectedViewer('IGV');
-        setShowToolSelector(true);
-      },
-      !entitiesSelected,
-      {
-        image: { width: 25, height: 25 },
-      }
+    return (
+      !snapshotName &&
+      h(
+        MenuTrigger,
+        {
+          side: 'bottom',
+          closeOnClick: true,
+          content: h(Fragment, [
+            h(
+              MenuButton,
+              {
+                onClick: () => {
+                  setSelectedViewer('IGV');
+                  setShowToolSelector(true);
+                },
+                disabled: !entitiesSelected,
+                tooltip: !entitiesSelected && 'Select rows to open in IGV',
+              },
+              'Open with IGV'
+            ),
+            h(MenuDivider),
+            h(
+              MenuButton,
+              {
+                onClick: () => {
+                  setSessionAction('load');
+                  setShowSessionModal(true);
+                },
+              },
+              'Load IGV Session'
```

Comment:
It's a bit awkward that "Load IGV Session" appears and is enabled even when I have no previously-saved sessions. Is it realistic to make this dynamic based on the existence of saved sessions?

Reply:
Fixed in https://github.com/DataBiosphere/terra-ui/pull/5393/commits/8d6c7f88e144bded15331d97bc8c7bb464acfc4e